### PR TITLE
Module.getVersion() -> Module.getSchemaVersion() and Module.getReleaseVersion()

### PR DESCRIPTION
**Pull request by labkey-adam**

### CODE REVIEW

File: core/src/org/labkey/core/query/ModulesTableInfo.java [hunk 1]
```diff
@@ -68,7 +68,8 @@ public void addColumns()
         nameCol.setURL(new StringExpressionFactory.URLStringExpression("${URL}"));
         nameCol.setURLTargetWindow("_blank");
 
-        addWrapColumn(getRealTable().getColumn("InstalledVersion"));
+        addTextColumn("ReleaseVersion").setScale(255);
```
Comment (labkey-jeckels) [inline core/src/org/labkey/core/query/ModulesTableInfo.java:71]: Think we should add a resolveColumn() method that resolves InstalledVersion for backwards compatibility with any existing queries? Can't imagine there are many that would be using it, so probably OK without it.

File: api/src/org/labkey/api/settings/AppProps.java [hunk 2]
```diff
@@ -95,7 +95,12 @@ static WriteableAppProps getWriteableInstance()
     @NotNull
     UsageReportingLevel getUsageReportingLevel();
 
-    String getLabKeyVersionString();
+    String getReleaseVersion();
```
Comment (labkey-jeckels) [inline api/src/org/labkey/api/settings/AppProps.java:101]: Add a note that these return the core module's info?

File: api/src/org/labkey/api/settings/AppProps.java [hunk 3]
```diff
@@ -95,7 +95,12 @@ static WriteableAppProps getWriteableInstance()
     @NotNull
     UsageReportingLevel getUsageReportingLevel();
 
-    String getLabKeyVersionString();
+    String getReleaseVersion();
+
+    /**
+     * Convenience method for getting the core schema version, guaranteed non-null.
+     */
+    double getSchemaVersion();
```
Comment (labkey-jeckels) [inline api/src/org/labkey/api/settings/AppProps.java:106]: Maybe say "... returning 0.0 instead of null" to be explicit about the behavior?

**Review by labkey-jeckels — APPROVED** [review]
A few suggestions, though none are mandatory.

File: core/src/org/labkey/core/query/ModulesTableInfo.java (hunk 1 above)
Comment (labkey-adam) [reply core/src/org/labkey/core/query/ModulesTableInfo.java:71]: Thanks... I thought about adding an "alias", but didn't remember how that was done. Done.

File: api/src/org/labkey/api/settings/AppProps.java (hunk 2 above)
Comment (labkey-adam) [reply api/src/org/labkey/api/settings/AppProps.java:101]: Done

File: api/src/org/labkey/api/settings/AppProps.java (hunk 3 above)
Comment (labkey-adam) [reply api/src/org/labkey/api/settings/AppProps.java:106]: Done

**Pull request merged**